src/main/java/com/google/jenkins/flakyTestHandler/plugin/HistoryAggregatedFlakyTestResultAction.java

```diff
@@ -175,12 +173,10 @@ public boolean getOnlyShowFlakyTests() {
   /**
    * Function to aggregate flaky stats over revisions
    */
-  public static final Function<Map<String, SingleTestFlakyStats>, SingleTestFlakyStats>
-      REVISION_STATS_MAP_TO_AGGREGATED_STATS = new Function<Map<String, SingleTestFlakyStats>,
-      SingleTestFlakyStats>() {
+  public static final Function<Map<String, SingleTestFlakyStats>, SingleTestFlakyStats> REVISION_STATS_MAP_TO_AGGREGATED_STATS = new Function<Map<String, SingleTestFlakyStats>, SingleTestFlakyStats>() {
     @Override
-    public SingleTestFlakyStats apply(
-        Map<String, SingleTestFlakyStats> revisionStatsMap) {
+    @Nonnull
+    public SingleTestFlakyStats apply(@Nonnull Map<String, SingleTestFlakyStats> revisionStatsMap) {
```

Comment:
why is `@Nonnull` added on these two lines? the windows CI is failing with `NullPointerException` pointing to these lines. If that is reverted, perhaps that will fix CI?

Reply:
This windows CI test was failing before I opened the PR for pipeline support. It seems to has nothing to do with it. You can see it on the other opened PR that the test is also throwing a NullPointerException.

Reply:
You're right; I forgot about that